tests/test_metrics/test_intervals_metrics.py

```diff
@@ -45,23 +130,49 @@ def test_width_metric_with_zero_width_quantiles(tsdataset_with_zero_width_quanti
         assert width_metric[segment] == expected_metric
 
 
-def test_width_metric_with_different_width_and_shifted_quantiles(tsdataset_with_different_width_and_shifted_quantiles):
-    ts_train, ts_test = tsdataset_with_different_width_and_shifted_quantiles
+@pytest.mark.parametrize(
+    "quantiles,lower_name,upper_name",
+    (
+        ((0.025, 0.975), "target_0.025", "target_0.975"),
+        ((), "target_0.025", "target_0.975"),
+        ((), "target_lower", "target_upper"),
+        ((0.1, 0.9), "target_lower", "target_upper"),
```

Comment:
How is this test case working? As I understood, there are only lower, upper, 0.025 and 0.975 columns.

Reply:
So, here we just try different ways to select interval.

1.  select the same interval with different methods
2. disable quantiles and use border names to select quantiles
3. disable quantiles and use border names to select other interval
4. quantiles point to non-existing interval, but names point to correct

This fixture contains both sets of intervals. It is done to simplify testing procedure, but somewhat artificial.